cupy/testing/_bundle.py

```diff
@@ -32,42 +13,24 @@ def make_decorator(test_case_generator):
     # The method generator is also a callable that receives an original test
     # method and returns a new test method.
 
-    def f(cases):
-        if isinstance(cases, _ParameterizedTestCaseBundle):
-            # The input is a parameterized test case.
-            cases = cases.cases
-        else:
-            # Input is a bare test case, i.e. not one generated from another
-            # parameterize.
-            cases = [_TestCaseTuple(cases, None, None)]
-
-        generated_cases = []
-        for klass, mod_name, cls_name in cases:
-            if mod_name is not None:
-                # The input is a parameterized test case.
-                # Remove it from its module.
-                delattr(sys.modules[mod_name], cls_name)
-            else:
-                # The input is a bare test case
-                mod_name = klass.__module__
+    def f(cls):
```

Comment:
Generally, a decorator takes a class as an input, creates a new class, and returns it. However, this decorator writes the generated new classes to the module, and replace the original class with `None`. I feel this behavior is a little bit tricky. Could you write a comment?

Reply:
Is `None` worse than the `_ParameterizedTestCaseBundle(cases)` instance?  The result has not been a class.

Reply:
I agree with your change. I just want you to leave a comment :)

Reply:
Added docstring of `testing.parameterize`.  (`return None` already has a comment.)